README.md

```diff
@@ -78,6 +78,8 @@ You can simply edit this list as if it were a document in vim.  `:wq` to save
 the new edits or `:q` to ignore the edits.  There is to save upon call to
 toggle if you prefer that way.
 
+You can also exit the list with `q` or `<ESC>`, which will also save your edits!
```

Comment:
does it actually save?  It seems like this would just quit, which wont actually save